sensei/svtkDataObjectToConduit.h

```diff
@@ -0,0 +1,45 @@
+/*=========================================================================
```

Comment:
should this file be in `utils/SVTK/IO/CatalystConduit` ?

Reply:
Good point. The only issue is that this has to be compiled conditionally on conduit being available, while SVTK is always compiled (mostly). Interestingly ascent does a similar type of conversion, so probably we could reuse this class in the ascent adaptor as well. @burlen Any suggestion or preferences?

Reply:
Another thought is that the CatalystConduit name is too ParaView centric for SENSEI. So I'll leave it where it is right now (no work) but I am open to moving it.

Reply:
> should this file be in `utils/SVTK/IO/CatalystConduit` ?

Nope, because SVTK is our fork of VTK, we don't want to be adding code in there.

Reply:
Sounds good. Note this code is also from VTK, but from parts you didn't forked over, and it includes some recent fixes /  additions I made to it. Do you plan to regularily update your SVTK fork?

Reply:
>  Do you plan to regularily update your SVTK fork?

No. The purpose of the fork is to create a stable data model so for our purpose it's better to not update VTK at all. Keeping new additions outside of the fork will ease the update process if and when that happens. Our fork isn't a complete fork, it's been minimized. Only Common (and dependencies) are in.  the other VTK components like Filtering, IO, Rendering, etc. aren't part of the fork.